### PR TITLE
🌱Add manifest collection after re-pivot

### PR DESCRIPTION
**Pull request by smoshiur1237**
This PR will add manifest collection after re-pivot

### CODE REVIEW

File: test/e2e/pivoting.go [hunk 1]
```diff
@@ -534,6 +534,22 @@ func rePivoting(ctx context.Context, inputGetter func() RePivotingInput) {
 		Intervals: input.E2EConfig.GetIntervals(input.SpecName, "wait-machine-running"),
 	})
 
+	By("Fetch manifest for ephemeral cluster after re-pivot")
```
Comment (kashifest) [inline test/e2e/pivoting.go:537]: target cluster?

Comment (smoshiur1237) [reply test/e2e/pivoting.go:537]: No, management cluster after pivot.

**Review by Sunnatillo — COMMENTED** [review]
LGTM

**Pull request merged**